src/autolabel/tasks/named_entity_recognition.py

```diff
@@ -230,11 +230,35 @@ def run_metrics(
         )
 
         results, _ = evaluator.evaluate()
-        # f1 score
+        # f1 score for strict match
         eval_metrics.append(
             MetricResult(
-                name=MetricType.F1,
-                value=results["exact"]["f1"],
+                name=MetricType.F1_STRICT,
+                value=results["strict"]["f1"],
+            )
+        )
+
+        # f1 score for exact match
```

Comment:
can we keep the exact f1 score to still be called F1 to stay backward compatible?

Reply:
Sure..